var/spack/repos/builtin/packages/openfoam-com/package.py

```diff
@@ -659,6 +663,10 @@ def __init__(self, spec, **kwargs):
         # spec.architecture.target is like `uname -m`
         target   = spec.architecture.target
 
+        # Platform detects as 'cray' - treat like 'linux'
+        if platform == 'cray':
+            platform = 'linux'
```

Comment:
@olesenm I'm using the cray compiler wrappers cc/CC, which (from my understanding) then run gcc/g++ with a bunch of extra options to locate MPI libraries, etc.  So it's fine that OpenFOAM directly runs g++ and specifies the MPI library locations itself, using the wmake rules `linux64Gcc`.